bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryDirectDataWriterHelper.java

```diff
@@ -176,8 +176,10 @@ private StreamWriter createStreamWriter(String writeStreamName) {
    */
   public void addRow(ByteString message) throws IOException {
     int messageSize = message.size();
```

Comment:
Could we just add 2 here? In messageSize?

Reply:
Done